de.metas.adempiere.adempiere/base/src/main/java/de/metas/logging/MetasfreshIssueAppender.java

```diff
@@ -80,26 +81,26 @@ public static final MetasfreshIssueAppender get()
 	 *
 	 * To be changed exclusively from {@link #temporaryDisableIssueReporting()}
 	 */
-	private final ThreadLocal<Boolean> issueReportingEnabled = new ThreadLocal<Boolean>()
+	private final ThreadLocal<Boolean> issueReportingEnabledInThread = ThreadLocal.withInitial(() -> Boolean.TRUE);
+
+	private volatile AtomicBoolean issueReportingEnabledGlobally = new AtomicBoolean(false);
+
+	public final void enableIssueReporting()
 	{
-		@Override
-		protected Boolean initialValue()
-		{
-			return Boolean.TRUE; // enabled by default
-		}
-	};
+		issueReportingEnabledGlobally.set(true);
```

Comment:
i think it would be nice to log.info about this

Reply:
sortof done..added logging to the listner that call the method.